packages/api/cms-api/src/dam/files/files.resolver.ts

```diff
@@ -48,6 +51,7 @@ export function createFilesResolver({ File, Scope: PassedScope }: { File: Type<F
             @InjectRepository("DamFile") private readonly filesRepository: EntityRepository<FileInterface>,
             @InjectRepository("DamFolder") private readonly foldersRepository: EntityRepository<FolderInterface>,
             private readonly contentScopeService: ContentScopeService,
+            private readonly fileValidationService: FileValidationService,
```

Comment:
I got the following error while testing:
<img width="1139" alt="Bildschirmfoto 2023-10-31 um 11 37 59" src="https://github.com/vivid-planet/comet/assets/13380047/ad1a4fa9-ae93-4158-a744-aad0265ebf2d">

I suppose the validation service isn't injected correctly. It worked for me after this change:

```suggestion
            @Inject(DAM_FILE_VALIDATION_SERVICE) private readonly fileValidationService: FileValidationService,
```

Reply:
yes, sorry